index.html

```diff
@@ -11,30 +11,190 @@
       href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
     />
   </head>
+
   <body>
     <header>
-      <a href="index.html" class="logo"><h1>Byiringiro Moise</h1></a>
+      <a href="index.html" class="logo">
+        <h1>Byiringiro Moise</h1>
+      </a>
       <span class="material-symbols-outlined"> menu </span>
     </header>
     <section class="profile">
-        <div class="introduction">
-<h2>Hey There. I'm Moise</h2>
-<h3>I'am a Software Developer</h3>
-<p>
-    I can help you build a product, feature or website look through some of my work and experience!
-    if you like what you see and have a project you need coded, don't hesitate to contact me.
-</p>
+      <div class="introduction">
+        <h2>Hey There. I'm Moise</h2>
+        <h3>I'am a Software Developer</h3>
+        <p>
+          I can help you build a product, feature or website look through some
+          of my work and experience! if you like what you see and have a project
+          you need coded, don't hesitate to contact me.
+        </p>
+        <ul class="social-media-icons">
+          <li>
+            <a href=""
+              ><img src="/items/Github-icon.png" alt="github icon"
+            /></a>
+          </li>
+          <li>
+            <a href=""><img src="/items/linkedin-icon.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="/items/Icon - Angelist.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="/items/Icon - Twitter.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="/items/M-icon.png" alt="" /></a>
+          </li>
+        </ul>
+      </div>
+     
+    </section>
+    
+ <!-- ------======================= work section mobile ================ ------>
+    <section class="work">
+        <div class="work-title">
+          <h4>My Recent Works</h4>
+        </div>
+        <section class="grid">
+          <div class="collection">
+            <div class="work1">
+               <img src="/items/img-for-first-project.png" alt="">
```

Comment:
Each project card requires an image. Even though you have added this one, the relative path is broken and so any of the images you have added are NOT showing:

**Your Project:**
![image](https://user-images.githubusercontent.com/92395484/192075656-b5fa3218-730f-4186-8b7e-257eaacc712e.png)

**Figma Template:**
![image](https://user-images.githubusercontent.com/92395484/192075680-b53fb484-e031-4fa5-8477-bd64328ef235.png)

Please make sure all of your images/icons are showing by correcting their path, for example, depending on the location of the images, you would probably like to add the source path like so: `./folder/image.png` :bulb: